### ISSUE TITLE
Faceoff No Keybind Functions

### ISSUE THREAD

**Issue by William0216** [Alexdoru/MWE#58]
In Faceoff, the keybinds do nothing. I have only tested /kill /ec and /tc binds

![image](https://github.com/user-attachments/assets/340b7f56-a796-4cfc-bd48-2a213a7f9d52)

**Comment by Alexdoru** [comment]
Yes game detection is broken in faceoff because I don't play faceoff and I don't plan to, I you can give me a screenshot of the scoreboard/Sidebar during a faceoff game, I can look into it

**Comment by William0216** [comment]
> Yes game detection is broken in faceoff because I don't play faceoff and I don't plan to, I you can give me a screenshot of the scoreboard/Sidebar during a faceoff game, I can look into it

Will do next time I'm in one

**Comment by William0216** [comment]
Faceoff Game Screenshot, once again, [F] is the most obvious difference compared to normal games
![image](https://github.com/user-attachments/assets/78148005-fe4a-4614-9d84-384032fa8ccc)
